### ISSUE TITLE
Refine CPC Definition Of Consensus 

### ISSUE THREAD

**Issue by bensternthal** [openjs-foundation/cross-project-council#1241]
Per comment on the travel fund pr in response to the following proposed workflow:

              

I looked at what we have in the Node.js project and it has the link to the wiki page but nothing else. My key point in the in the recent working group meeting is that the "consensus" must represent that of the CPC, not just the subset of CPC members who could make a particular meeting. That requires that people not in attendance have the opportunity to object before the decision is considered final. I think we might want to update the description in https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#consensus-seeking-process to reflect that as well. For example, if two people show up at a CPC meeting they are not able to make a decision on behalf of all of the CPC.  Further even if 51% of the people show up at a meeting the other 49% of the members should still have an opportunity to object before the decision is considered final.

_Originally posted by @mhdawson in https://github.com/openjs-foundation/cross-project-council/pull/1230#discussion_r1456605087_

**Comment by tobie** [comment]
I'd remove the travel-fund label and push it as a broader governance issue. This is how the CPC operates anyway, so this is more about making paper governance better spell-out actual governance.

**Comment by tobie** [comment]
Following-up on our conversation in the CPC working session today, I'd suggest [adapting the PR merge request process](https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#merging-prs-into-this-repository) for  travel fund requests. The PR merge process says the following:

> Pull requests that do not change the charter or governance of the CPC can be merged into this repository provided the following conditions have been met:
>
> - There are no outstanding objections
> - There are two approvals by CPC members
> - The PR has been open for at least 72 hours
>
> Pull requests that change governance of the CPC (including approving changes to Working Groups) must meet the following conditions in addition to the ones listed for regular PRs.
> 
> - The PR has been open for at least 14 days OR consensus is reached in a meeting with quorum of voting members.

I'd suggesting the following for approving travel fund requests:

> Funding requests that have a rubric score of _x_ or higher are approved provided the following conditions have been met:
>
> - There are no outstanding objections
> - There are two approvals by CPC members
> - The email has been sent for at least 72 hours
>
> Funding requests that have a rubric score lower than _x_ must meet the following conditions in addition to the ones listed above:
> 
> - The email has been open for at least 14 days OR consensus is reached in a meeting with quorum of voting members.

**Comment by ljharb** [comment]
Does that mean the rubric score would be publicly calculated and posted?

**Comment by joesepi** [comment]
@ljharb - The rubric score would not be public. None of this would be happening in Github. But the rubric could be shared in the private email to CPC members and in the travel fun review that should happen on a regular basis in a private CPC section of a regular CPC meeting.

I think the part that gets tricky @tobie with your suggestion is if we have more applicants than money, which I believe will be bucketed by quarter. In this instance, there may need to be some consensus and no objections. I'm not saying there incompatibility here in what you are proposing, but I think this thinking needs to be more incorporated into the potential flow of decision making around these requests, including when there are more requests than money.

**Comment by tobie** [comment]
We're going to have to "solve not enough money" with a discussion. Technically, it's just a -1.

**Comment by joesepi** [comment]
One thing to point out (technically two or three things to point out): we have consensus "defined" in both the Charter and the Governance. 🤦 

- In the [Governance](https://github.com/openjs-foundation/cross-project-council/blob/main/governance/GOVERNANCE.md#consensus-seeking-process)
- In the [Charter under Decision Making](https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-9-decision-making), but also in the [Charter under "Voting"](https://github.com/openjs-foundation/cross-project-council/blob/main/CPC-CHARTER.md#section-10-voting)
And also

**Comment by tobie** [comment]
> One thing to point out (technically two or three things to point out): we have consensus "defined" in both the Charter and the Governance. 🤦

Yeah. This is why it's valuable to separate policy (high-level, non-specific, belongs in the charter) from process (the implementation of the policy, which belongs in governance.md or similar).

**Comment by mhdawson** [comment]
We discussed in the CPC meeting today and sounds like what I think we currently follow/should follow is generally accepted so I volunteered to write up a paragraph that better describes that.

**Comment by mhdawson** [comment]
Pull request - https://github.com/openjs-foundation/cross-project-council/pull/1281

**Comment by mhdawson** [comment]
If we agree on 1281 for the Governance I'll then submit a PR to update the charter to be more consistent with what gets landed.

**Comment by mhdawson** [comment]
closing since https://github.com/openjs-foundation/cross-project-council/pull/1281 landed.
